Makefile

```diff
@@ -191,6 +192,10 @@ check:
 generate:
 	@hack/generate.sh ./cmd/... ./extensions/... ./pkg/... ./plugin/... ./test/...
 
+.PHONY: generate-extensions-crds
+generate-extensions-crds:
+	@controller-gen crd paths=./pkg/apis/extensions/... output:crd:dir=./dev/extensions-crds output:stdout
```

Comment:
Nice, also wanted to propose using controller-tools for that :) 
We could also generate the CRDs in `charts/seed-bootstrap/templates/extensions` again, right?
Should we move this to `hack/update-codegen.sh` or use a `go:generate` comment in `pkg/apis/extensions`?

Reply:
> We could also generate the CRDs in charts/seed-bootstrap/templates/extensions again, right?

WDYM? I generated them and added/updated the open API validation spec. Did I forget something?

> Should we move this to hack/update-codegen.sh or use a go:generate comment in pkg/apis/extensions?

This won't be easily possible (we can't just blindly take what was generated from this CLI tool), especially now that we have `if-end` expressions in the CRDs.

Reply:
> This won't be easily possible (we can't just blindly take what was generated from this CLI tool), especially now that we have if-end expressions in the CRDs.

Yeah, sorry. Forgot about the `if`s in the CRDs.
Then you can ignore my comment.

Reply:
OK, thanks!